### ISSUE TITLE
alexa-remote-event payload has been changed - any ideas?

### ISSUE THREAD

**Issue by Spunckie** [586837r/node-red-contrib-alexa-remote2#130]
until yesterday the Alexa-remote-event works fine. In msg.payload.description.summary found the text that Alexa said. But now the field is empty. I rebooted my Alexa-Devices and Node Red. 
Any Ideas how to solve this problem? Thanks .

**Comment by ethan-ingalls** [comment]
I'm having the same issue but worse. No output is given at all

**Comment by quadhammer** [comment]
Did you have any luck fixing this?  It's happened to me just recently, and I think I updated the node just recently as well.  I will try restoring to an earlier version, and see if that works, but I'm not having much luck trying to do that at the moment.

**Comment by CursedDog19** [comment]
Yes I am pretty sure that the bug resolved itself within the next few days. Maybe try restarting the host server and reinstalling the nodered package?

**Comment by CursedDog19** [comment]
> Did you have any luck fixing this? It's happened to me just recently, and I think I updated the node just recently as well. I will try restoring to an earlier version, and see if that works, but I'm not having much luck trying to do that at the moment.

Yes I am pretty sure that the bug resolved itself within the next few days. Maybe try restarting the host server and reinstalling the nodered package?

**Comment by quadhammer** [comment]
Mine resolved itself too...without reinstalling.  Hope it was just a once off.
